python/tvm/script/tir/__init__.pyi

```diff
@@ -464,14 +464,19 @@ class uint32(PrimExpr):
 class uint64(PrimExpr):
     def __init__(self: uint64, imm: Union[PrimExpr, int]): ...
 
+# use typing.Literal instead for python 3.8 or higher
+from typing_extensions import Literal
```

Comment:
I think it's be better to avoid extra dependency. We can check the sys version and upgrade to 3.8 sometime in the future
```
if sys.version_info >= (3, 8):
  SpecialFloatLiteral = ...
else:
  SpecialFloatLiteral = str
```

Reply:
Sounds good. This is changed. Since `Enum` seems like an overkill I just keep the `SpecialFloatLiteral = str` here instead of `class SpecialFloatLiteral(Enum)`